catalog/cataloger_list_commits.go

```diff
@@ -71,10 +71,11 @@ func (c *cataloger) ListCommits(ctx context.Context, repository, branch string,
 	return commits, hasMore, err
 }
 
-func convertRawCommits(rawCommits []*commitLogRaw) []*CommitLog {
+func convertRawCommits(rawCommits []commitLogRaw) []*CommitLog {
 	commits := make([]*CommitLog, len(rawCommits))
 	for i, commit := range rawCommits {
-		commits[i] = convertRawCommit(commit)
+		copy := commit
```

Comment:
why do we need copy? we do not use the pointer - if we are passing references I prefer convertRawCommit to take a struct and not a pointer so we will not remember to make a copy